### ISSUE TITLE
4.1.0 on npm

### ISSUE THREAD

**Issue by Gerschtli** [mledoze/countries#478]
Can you please release 4.1.0 on npm registry?

**Comment by mledoze** [comment]
The 4.1.0 version is released https://www.npmjs.com/package/world-countries/v/4.1.0

(sorry for the delay)

**Comment by Gerschtli** [comment]
@mledoze The package.json contains:
```json
    "exports": {
        "import": "./index.mjs",
        "require": "./index.cjs"
    },
```
But these files are not included in the package, therefore it is not usable currently.

**Comment by mledoze** [comment]
You're right, I will fix that, hopefully today.

**Comment by mledoze** [comment]
Fixed in 4.1.1 version https://www.npmjs.com/package/world-countries/v/4.1.1
